### ISSUE TITLE
LNDC RPC Client does not have a Close option

### ISSUE THREAD

**Issue by gertjaap** [mit-dci/lit#397]
It desperately needs it:

```
lit-af# ls -a
entered command: ls -a
	Peers:
0 172.18.0.2:49144 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44230 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44590 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44652 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47538 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48364 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48494 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47986 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:49130 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44732 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44926 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45002 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45164 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45424 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46442 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48938 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47418 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47528 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47940 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48016 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48238 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48350 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46356 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47874 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44812 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44848 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44878 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45132 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45528 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46216 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44218 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45562 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46070 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46834 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:49176 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45568 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46334 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48000 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48968 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44672 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46796 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46924 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48266 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48398 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45950 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47218 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47384 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48526 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:48542 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44908 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45264 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45668 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47586 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45860 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45960 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46154 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:46628 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47050 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47272 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:44896 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45360 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:45796 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
0 172.18.0.2:47966 (ln1azfhjdevwr6v0ger5ccgjf4stryfqcnp3ndx52)
```

**Comment by delbonis** [comment]
I think this is fixed in #384.

**Comment by Varunram** [comment]
Merged in https://github.com/mit-dci/lit/pull/416/commits/326dad12a91e23fd255f4bcd40d8c1529c2bd786
